src/components/Semester.js

```diff
@@ -62,6 +62,11 @@ class Semester extends Component {
                 variant="outlined" color="primary" style={{margin: 10}}>
                 Get Schedule
               </Button>
+              <Button component={Link} 
+                      to={{pathname:'/student/add'}} 
```

Comment:
If you don't have any data to pass to the router link, you can just use a string path instead of an object.

`to="/student/add"`